### DISCUSSION TITLE
Can Area site's 12-step-meeting-list plug-in read feed from District sites' plug-ins?

### DISCUSSION THREAD

**Discussion by BillCinVT** [code4recovery/12-step-meeting-list#1207] [Q&A]
Hi, all! Thanks, huge thanks for this wonderful plug-in and its community. 
I apologize if I'm not looking in the right way, but I've searched the forums in vain.
I know that the Meeting Guide app "reads" the feed of our Area 12-step-meeting-list plug-in, automatically updating info about our Area's meetings. 
**Can our Area 12-step-meeting-list plug-in be made to "read" the feed from District plug-ins?**
I realize that's probably an ignorant question, but I don't know how to describe the online processes or other programs that might be needed to effect this. 
Many thanks,
BillCinVT
District 5 Meeting List Chair

**Comment by anchovie91471** [comment]
So your districts have their own website with 12 Step Meeting List installed?

**Reply by tim-rohrer** [reply]
Hey @BillCinVT, As Anthony is alluding to, if you have TSML on the district sites, doing what you're asking for is quite simple. 

1. On the district sites, make the meeting feed public. Meetings->Settings->Feed Management->Feed Sharing->Open
2. Note the Public Data Source URL that becomes available.
3. Use that URL on the Area site to import JSON feeds. Meetings->Import & Export.

Check out [this howto](https://code4recovery.org/how-to/how-to-list-meetings-from-multiple-organizations) for more information.

**Comment by BillCinVT** [comment]
Yes. For example, Vermont District 5 is represented by uppervalleyaa.com,
using 12-step-meeting-list, and Area 70, VT, found on aavt.org, also uses
12-step-meeting-list.

It seems pretty common for Districts to have their own sites. Sometimes
just cities or communities within Districts.
In the case of District 5, which I serve, a separate site is a good idea
because our "Upper Valley" region straddles the Connecticut River. People
living here often attend meetings in both New Hampshire and Vermont. Area
70 won't list NH meetings, so we do.
